src/lbaf/IO/lbsVTDataWriter.py

```diff
@@ -62,7 +64,7 @@ def __create_tasks(self, rank_id, objects, migratable):
             task_data = {
                 "entity": {
                     "home": rank_id,
-                    "id": o.get_id(),
+                    "seq_id": o.get_id(),
```

Comment:
Similar to the reader, if LBAF received a vt `id` from the JSON, then it should write out the `id` field here as well (and not `seq_id`).

It may help to add a flag in the `Object` class to keep track of whether or not the initial identifier is an `id` or `seq_id`.

Reply:
@cwschilly I think this is not a good idea if input files contain both id and seq_id we should keep the writer able to write the 2 values so we need to keep both in memory